### DISCUSSION TITLE
Tasmota MQTT HDMI Display

### DISCUSSION THREAD

**Discussion by dacCR** [arendst/Tasmota#21071] [Q&A] [answered]
I've read that Tasmota has an HDMI method/capability. 
Would it be possible to have a Tasmota device that has an HDMI output where I can use MQTT to send text and images to the output?

**Comment by sfromis** [comment]
Tasmota has no HDMI capability, except for [HDMI CEC](https://tasmota.github.io/docs/HDMI_CEC/) to control a TV (or other devices) responding to this protocol, on a similar level as a remote control.

**Reply by barbudor** [reply, marked as answer]
Indeed you won't be able to send an image, but on a TV supporting CEC, i believe there are CEC commands to display text overlays

**Reply by dacCR** [reply]
Thank you. I wasn't sure that I understood and do now.
